examples/rust-flake-project-with-extra-dependency/.gitignore

```
@@ -0,0 +1,2 @@
+/target
+/.direnv
```

```suggestion
/.direnv
.extras
```
Perhaps we should include the extra sources as ignored since nix produces it?